modules/ROOT/pages/prereq-software.adoc

```diff
@@ -18,10 +18,11 @@ Before you install Anypoint Platform PCE, your infrastructure team must review e
 === Verify Linux Distribution
```

Comment:
The section above where it says "Before you install Anypoint Platform PCE,...", the "you" could be misleading since it might imply that the customer is able to install PCE by himself when they actually need services to install it